ReactSkia/views/common/RSkImageCacheManager.h

```diff
@@ -13,20 +13,32 @@
 #define SKIA_CPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 #define SKIA_GPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 
+using namespace std;
+
 namespace facebook {
 namespace react {
+namespace ImageCacheManager {
+
+typedef facebook::better::map <string,sk_sp<SkImage>> ImageCacheMap;
+
+class RSkImageCacheManager {
+ public:
+  RSkImageCacheManager();
+  ~RSkImageCacheManager();
+  static void init();
+  static RSkImageCacheManager& getImageCacheManagerInstance();
+  sk_sp<SkImage> findImageDataInCache(const char* path);
+  void imageDataInsertInCche(const char* path,sk_sp<SkImage> imageData);
```

Comment:
Correct the typo in function name - imageDataInsertInCache   => letter 'a' is missing.